contracts/contracts/ccip/ccipsend_executor/errors.tolk

```diff
@@ -1,3 +1,4 @@
-tolk 1.0
-
-const CCIPSEND_EXECUTOR_ERROR_STATE_NOT_EXPECTED = 500;
\ No newline at end of file
+enum CCIPSendExecutor_Error {
```

Comment:
imo no need to split out a <10 loc file to follow a structural convention - we could also support a simpler structure where when not necessary to split out the errors (not huge amount of errors), and we keep the errors as part of `types.tolk` or even a single file simple contract :)

I wouldn't want to split a 40 loc mock contract to 4x files.